src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1744,6 +1744,13 @@ static void updateAttrInNotifyCer
 
   /* Reached this point, it means that it is a new attribute (APPEND case) */
   ContextAttribute* caP = new ContextAttribute(targetAttr, useDefaultType);
+
+  if (caP->compoundValueP)
+  {
+    // The ContextAttribute constructor steals the compound, but in this case, it must be cloned
+    targetAttr->compoundValueP = caP->compoundValueP->clone();
```

Comment:
Is this leak-free?

(Every time I see a clone() method, an alarm about leaks triggers in my mind :)